eventlet/websocket.py

```diff
@@ -692,7 +698,7 @@ def _recv_frame(self, message=None):
         received = 0
         if not message or opcode & 8:
             decoder = self.UTF8Decoder() if opcode == 1 else None
-            decompressor = self._get_permessage_deflate_dec()
+            decompressor = self._get_permessage_deflate_dec(rsv123 & 0x04)
```

Comment:
please leave a comment about magic number mask

Reply:
A similar magic number thing happens around [line 657](https://github.com/costasgambit/eventlet/blob/ab2300a5eded021c6b4dd07bcf07f032526d4557/eventlet/websocket.py#L657) with shifts. Would it be better add `rsv1 = rsv123 & 4` or `rsv1 = rsv123 >> 2` somewhere around that part and use just `rsv1` later? Possibly adding a comment with a link to [Section 5.2 of RFC6455](https://tools.ietf.org/html/rfc6455#section-5.2) or copying the first 8 bits of the diagram in that section.

Reply:
Please add link to RFC at first encounter of `rsv...` name.